src/components/AppNavigationCaption/AppNavigationCaption.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+	<li class="app-navigation-caption--item">
+		{{ text }}
+	</li>
+</template>
+
+<script>
+
+export default {
+	name: 'AppNavigationCaption',
+	props: {
+		text: {
+			type: String,
+			required: true
+		}
+	}
+}
+</script>
+<style scoped>
+.app-navigation-caption--item {
+	font-weight: bold;
+	color: var(--color-text-maxcontrast);
+	line-height: 44px;
```

Comment:
Please use the $clickable-area variable :)
https://github.com/nextcloud/nextcloud-vue/blob/c886d5c852ba89580c4ab6dd098a9fe0591197c6/src/assets/variables.scss#L26

```suggestion
	line-height: $clickable-area;
```